pkgs/servers/sql/mariadb/galera/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, buildEnv
+, asio, boost, check, openssl, scons
+, version, sha256, ...
+}:
+
+let
+  galeraLibs = buildEnv {
+    name = "galera-lib-inputs-united";
+    paths = [ openssl.out boost check ];
+  };
+
+in stdenv.mkDerivation rec {
```

Comment:
move pname into the let block and inherit it and then you can drop the rec on the attrSet.

Reply:
@disassembler that?
```
diff --git a/pkgs/servers/sql/mariadb/galera/default.nix b/pkgs/servers/sql/mariadb/galera/default.nix
index 3fb6ebfbfc9..263a21983e2 100644
--- a/pkgs/servers/sql/mariadb/galera/default.nix
+++ b/pkgs/servers/sql/mariadb/galera/default.nix
@@ -4,13 +4,14 @@
 }:

 let
+  pname = "mariadb-galera";
   galeraLibs = buildEnv {
     name = "galera-lib-inputs-united";
     paths = [ openssl.out boost check ];
   };

-in stdenv.mkDerivation rec {
-  pname = "mariadb-galera";
+in stdenv.mkDerivation {
+  inherit pname;
   inherit version;

   src = fetchFromGitHub {
```